p2p/p2p.go

```diff
@@ -120,7 +131,14 @@ func NewService(conf *Config, msgChan chan<- Message) (*Service, error) {
 		msgChan:        msgChan,
 	}
 
+	s.statusMessagesRec = make(map[string]bool)
+	s.blockReqRec = make(map[string]bool)
+	s.blockRespRec = make(map[string]bool)
+	s.blockAnnounceRec = make(map[string]bool)
+	s.txMessageRec = make(map[string]bool)
+
 	h.SetStreamHandler(ProtocolPrefix, s.handleStream)
+	h.SetStreamHandler(ProtocolPrefix2, s.handleBroadcastStream)
```

Comment:
there is only one protocol prefix for a network, and there should only be one stream handler for it